httpbakery/agent/agent.go

```diff
@@ -78,11 +70,38 @@ type agentLogin struct {
 	PublicKey *bakery.PublicKey `json:"public_key"`
 }
 
-// agentResponse defines the structure of an agent login response.
+// agentResponse contains the response to an agent login attempt.
 type agentResponse struct {
 	AgentLogin bool `json:"agent_login"`
 }
 
+var ErrNoAgentLoginCookie = errgo.New("no agent login cookie found")
```

Comment:
This error should probably returned by VisitWebPage if it can't find the cookie instead of http.ErrNoCookie, there are no clients that will rely on that yet.